app/views/migration/_form.html.erb

```diff
@@ -82,6 +82,10 @@
       <% end %>
     <% end %>
 
+    <% if @permitted_aasm_events %>
```

Comment:
This could be more specific, or add another nested `if`, and check there's actually events to change to. On deleted stories you get an empty dropdown. In that case I reckon we should just show the current state and no dropdown.